llm/server.go

```diff
@@ -346,6 +346,7 @@ func NewLlamaServer(gpus gpu.GpuInfoList, model string, ggml *GGML, adapters, pr
 		s.cmd.Env = os.Environ()
 		s.cmd.Stdout = os.Stdout
 		s.cmd.Stderr = s.status
+		subprocessSettings(s.cmd)
```

Comment:
could we do something like

```
if runtime.GOOS == "windows" {
  ...
  cmd.SysProcAttr.CreationFlags = CREATE_DEFAULT_ERROR_MODE
}
```

which might be a bit cleaner than the platform-specific noop functions

Reply:
This isn't possible since these symbols don't exist on other platforms - it will result in a compile error.

Reply:
Gotcha. Could we make the function return a `*syscall.SysProcAttr` then and assign it here (vs making a mutating function)?

Reply:
Or even a constant we can assign

```
cmd.SysProcAttr = LlamaServerSysProcAttr
```